pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -1122,4 +1129,57 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+
+	err = InfraSAs.addServiceAccount(
+		InfraTemplateInstanceControllerServiceAccountName,
+		authorizationapi.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: TemplateInstanceControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					APIGroups: []string{templateapi.GroupName},
+					// "impersonate" is required for the API server to accept updates to
+					// TemplateInstance objects where the requester username is not the
+					// API caller.
+					Verbs:     sets.NewString("get", "list", "watch", "update", "impersonate"),
```

Comment:
The controller updates status only.  At the moment, things are set up such that you must be the requester or have "impersonate" to edit any part of the object, hence "impersonate" is required here.  Would it be better to allow anyone to change the status?  Or is it somehow possible to restrict things so that just the controller can update status, and this can be done without "impersonate"?